README.md

```diff
@@ -71,6 +71,47 @@ that may be enabled to help troubleshoot. The `qsc` command-line compiler makes
 crate [env_logger](https://docs.rs/env_logger/latest/env_logger/), which enables logging via
 environment variables, for example `RUST_LOG=debug ./target/release/qsc ./samples/Grover.qs`.
 
+## Citation
+
+If you use Q#, Azure Quantum Development Kit, and Azure Quantum Resource Estimator, please cite as follows:
```

Comment:
nitpick: "or" sounds more natural here, since using any one of these could generate a citation.
```suggestion
If you use Q#, Azure Quantum Development Kit, or Azure Quantum Resource Estimator, please cite as follows:
```